logic/telemetry.go

```diff
@@ -13,8 +13,11 @@ import (
 	"golang.org/x/exp/slog"
 )
 
-// flags to keep for telemetry
-var isFreeTier bool
+var (
+	// flags to keep for telemetry
+	isFreeTier      bool
+	telServerRecord = models.Telemetry{}
```

Comment:
telmetry record gets updated as well, this variable won't be in sync with DB

Reply:
done,  cache updated when telemetry is updated.